packages/ozone/src/api/moderation/createReport.ts

```diff
@@ -0,0 +1,34 @@
+import { Server } from '../../lexicon'
+import AppContext from '../../context'
+import { getReasonType } from './util'
+import { subjectFromInput } from '../../mod-service/subject'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.moderation.createReport({
+    // @TODO anonymous reports w/ optional auth are a temporary measure
+    auth: ctx.authOptionalAccessOrRoleVerifier,
```

Comment:
I don't think we _necessarily_ need to support role-based auth here since admins could go straight to `emitModerationEvent()`.  In #1969 this is actually utilized a bit.